app.config.ts

```diff
@@ -33,19 +33,26 @@ export default ({ config }: ConfigContext): ExpoConfig => ({
   },
   version: appBuildNumbers.expo.version,
   assetBundlePatterns: ["**/*"],
+  plugins: [
+    "@react-native-firebase/app-check",
+    ["expo-build-properties", { ios: { useFrameworks: "static" } }],
```

Comment:
I don't think this is doing anything since iOS is not in an expo Continuous Native Generation workflow, but need to verify by removing it and building in higher envs

Reply:
Correct, I can guarantee it is not doing anything

Reply:
![i-guarantee-it](https://github.com/user-attachments/assets/b00eafba-8009-4cc8-9c11-d25314e47a38)